src/js/tabs/ports.js

```diff
@@ -351,50 +350,48 @@ TABS.ports.initialize = function (callback, scrollPosition) {
 
    function on_save_handler() {
         analytics.sendChangeEvents(analytics.EVENT_CATEGORIES.FLIGHT_CONTROLLER, self.analyticsChanges);
-       self.analyticsChanges = {};
+        self.analyticsChanges = {};
 
         // update configuration based on current ui state
         FC.SERIAL_CONFIG.ports = [];
 
-        var ports_e = $('.tab-ports .portConfiguration').each(function (portConfiguration_e) {
-
-            var portConfiguration_e = this;
-
-            var oldSerialPort = $(this).data('serialPort');
+        $('.tab-ports .portConfiguration').each(function (portConfiguration) {
+            const portConfig = portConfiguration;
```

Comment:
This assignment does not seem to be making sense - we can just use `portConfiguration` below.